Documentation/markdown/SetupHelp.md

```diff
@@ -80,3 +83,4 @@ Before you can proceed to the Installation of **OpenInfraPlatform**, check, if t
 2. If you are using point clouds *build* project **OpenInfraPlatform.PointCloudProcessing**.
 3. *Build* project **OpenInfraPlatform.Core**.
 4. *Build* project **OpenInfraPlatform.UI**.
+5. *Build* all projects staged in the **Copy** project folder. **NOTE:** execute **Get_OKLABI.cmd** in the source folder (e.g. C:\dev\Open-Infra-Platform\external\) to prevent an error during the build process. 
```

Comment:
To point 5: After the execution of **Get_OKLABI.cmd**, it's necessary to configure and generate once again in CMake.
Sorry, I missed this information yesterday.
[onec again, now my comment should be at the right place - including the last changes in the code]

Reply:
Done